crates/cli/Cargo.toml

```
@@ -26,6 +25,7 @@ rouille = { version = "3.0.0", default-features = false }
 serde = { version = "1.0", features = ['derive'] }
 serde_derive = "1.0"
 serde_json = "1.0"
+ureq = { version = "2.7", default-features = false, features = ["gzip"] }
```

```suggestion
ureq = { version = "2.7", default-features = false, features = ["brotli", "gzip"] }
```
Probably irrelevant, but why not while we are there.

---

Agreed. When not turning on this feature, it is possible that we received an **un**decoded binary value as a HTTP response instead of an encoding failure error.